redshift_connector/cursor.py

```diff
@@ -239,9 +244,31 @@ def executemany(self: "Cursor", operation, param_sets) -> "Cursor":
         self._row_count = -1 if -1 in rowcounts else sum(rowcounts)
         return self
 
+    def insert_data_bulk(self, filename, url, name, links, url_index, name_index):
+        """
+
+           runs a single bulk insert statement into the database.
+
+        """
```

Comment:
Could the method doc string please be expanded to provide descriptions and expected types for each parameter